Assets/MRTK/Tests/EditModeTests/SpatialAwareness/SpatialAwarenessSystemTests.cs

```diff
@@ -50,19 +50,6 @@ public void TestSpatialAwarenessSystemExists()
             Assert.IsTrue(MixedRealityToolkit.Instance.IsServiceRegistered<IMixedRealitySpatialAwarenessSystem>());
         }
 
-        [Test]
```

Comment:
Not super familiar with this piece of code, any reason we're removing this test?

Reply:
Not sure if this is the exact reason @wiwei hit, but from my experience this test will fail on UWP since we _do_ have a spatial awareness provider configured for that build type.
It seems our CI / PR validation tests run on standalone, so there's no valid mesh provider and this should be empty.

Reply:
Yep, this is exactly it - I hit this with the test and I from what I can see, it doesn't seem like a valid test to me - i.e. what this is verifying/validating is that for ANY platform that we have, we will never have a default spatial awareness system set up by default. I don't think that this statement is true.

Reply:
I'm happy that you asked!